test/e2e/serial/config/config.go

```diff
@@ -164,10 +165,11 @@ func CheckNodesTopology(ctx context.Context) error {
 		return fmt.Errorf("Following nodes have incoherent info in KubeletConfig/NRT data:\n%s\n", string(prettyMap))
 	}
 
-	snnPodLevelNRTs := e2enrt.FilterTopologyManagerPolicy(nrtList.Items, nrtv1alpha2.SingleNUMANodePodLevel)
+	//TODO - Can look just for single-numa-node policy whatever the scope
```

Comment:
in some cases yes. Some testcases however
1. need to run against `single-numa-node` policy but
2. make different logic (by design) depending on `scope`.

So somehow we will need to check both (or to rewrite the tests)

Reply:
Well, in this particular case I think we can do so as we are only counting the number of nodes with the wanted topology policy.

Reply:
yes, in this case I think we can count only the nodes per topology and error out if that number is `< minNumberOfNodesWithSameTopology`

Instead of two check we can just have indeed

```
...
// TODO was here
...
snnContainerLevelNRTs := e2enrt.FilterByTopologyManagerPolicy(nrtList.Items, intnrt.SingleNUMANode)
numSnnContainerLevelNRTs := len(snnContainerLevelNRTs)
if numSnnPodLevelNRTs < minNumberOfNodesWithSameTopology {
...
```